src/ostorlab/apis/assets.py

```diff
@@ -0,0 +1,189 @@
+"""Create Asset via an API Request."""
+import json
+from typing import Dict, Optional
+
+from ostorlab.apis import request
+from ostorlab.assets import asset as base_asset
+
+
+class CreateAssetAPIRequest(request.APIRequest):
```

Comment:
This class is complex enough compared to the other request that it warrants a unit test.